src/ralph/accounts/views.py

```diff
@@ -220,6 +225,17 @@ def get_context_data(self, **kwargs):
             request=self.request,
         )
 
+        context['simcard_list'] = AssignedSimcardsList(
+            self.get_simcard_queryset(),
+            [
+                ('phone_number', _('Phone Number')),
+                ('card_number', _('Card Number')),
+                ('pin1', _('PIN 1'))
```

Comment:
Perhaps puk1 and carrier name could be useful for users as well.

Reply:
I agree, I bet pin is not something users will have to check for, especially that users will be likely to change it on the physical card. Most likely it will be puk that they will need, after entering wrong pin 3 times.